bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/OpenWeatherMapBindingConstants.java

```diff
@@ -87,6 +87,9 @@ public class OpenWeatherMapBindingConstants {
     public static final String CHANNEL_TIME_STAMP = "time-stamp";
     public static final String CHANNEL_SUNRISE = "sunrise";
     public static final String CHANNEL_SUNSET = "sunset";
+    public static final String CHANNEL_MOONRISE = "moonrise";
+    public static final String CHANNEL_MOONSET = "moonset";
+    public static final String CHANNEL_MOON_PHASE = "moon_phase";
```

Comment:
Please follow the naming convention used by this binding (and sidenote: required for new bindings: https://www.openhab.org/docs/developer/guidelines.html#naming-convention):

```suggestion
    public static final String CHANNEL_MOON_PHASE = "moon-phase";
```